apps/goldsky-subgraph/README.md

```diff
@@ -0,0 +1,69 @@
+# Index & Query Berachain Data with Goldsky
+
+An example using the [Goldsky](https://goldsky.com/) SDK to index Berachain blockchain data. Specifically, this repo provides subgraph code for indexing user ERC20 token balances.
+
+👉 Try the [demo](https://api.goldsky.com/api/public/project_clteviu2hajla01r51uil7cp5/subgraphs/erc20-subgraph/1.0.0/gn)
+👉 Read the Goldsky [developer docs](https://docs.goldsky.com/introduction)
+
+## Requirements
+
+- Node js > 20
+- pnpm (or another prefered package manager)
+
+## Quick Setup
+
+### Step 1 - Install dependencies
+
+```
+# FROM: ./goldsky-subgraph
+
+pnpm install;
+```
+
+### Step 2 - Set up Goldsky
+
+1. Create an account at app.goldsky.com
+2. Create an API key on the Settings page
+3. Install the Goldsky CLI:
+
+```
+curl https://goldsky.com | sh
+```
+
+4. Log in with the API key created earlier:
+
+```
+goldsky login
+```
+
+### Step 3 - Deploy Subgraph
+
+```
+# FROM: ./goldsky-subgraph
+
+goldsky subgraph deploy erc20-subgraph/1.0.0 --path .
+```
+
+### Step 4 - Query the Subgraph
+
+After your subgraph has finished indexing, navigate to your public GraphQL link and try the following query:
```

Comment:
Is there a public link to this maybe to give the user an idea of what this looks like.

Maybe even a screenshot would help.